pkgs/development/libraries/gnu-config/default.nix

```diff
@@ -23,6 +23,10 @@ in stdenv.mkDerivation {
   pname = "gnu-config";
   version = "2023-09-19";
 
+  patches = [
+    ./support-uefi-triples.patch
+  ];
+
```

Comment:
I'm flattered that you included my patch but since it hasn't been merged upstream I don't think we should do this yet.

Instead I've added `*-*-uefi` to the exception-list: https://github.com/NixOS/nixpkgs/pull/235230/commits/54ddd6e1985689752a932247063863d3fa8f43e3

If `gnu-config` merges the patch then we can just drop the exception.

Reply:
Pushed a17e894deec18892c1c930f293733517dfdb51cf to this branch to drop the patch.